packages/jest-diff/src/index.ts

```diff
@@ -160,6 +162,7 @@ namespace diff {
   export type DiffOptions = JestDiffOptions;
 }
 
-diff.getStringDiff = getStringDiff;
```

Comment:
Breaking, correct? Just mentioning because the changelog entry does not say so

Reply:
Yes, I had hoped to beat the minor because that export is only for first time in 24.9.0 :(

Reply:
let's just pretend it was never public api. Nobody will be depending on it yet.

We might also not have a release until 25, at which point it doesn't matter anyways